docs/architecture/grpc-for-wcf-developers/protobuf-maps.md

```diff
@@ -47,3 +47,5 @@ For more information about Protobuf, see the official [Protobuf documentation](h
 >[!div class="step-by-step"]
 >[Previous](protobuf-enums.md)
 >[Next](wcf-services-to-grpc-comparison.md)
+
+[map-field]: https://developers.google.cn/protocol-buffers/docs/reference/csharp/class/google/protobuf/collections/map-field-t-key-t-value-
```

Comment:
You should add the link on line 46. I'm pretty sure it breaks how the book is rendered if you add something below the footer.

Reply:
Ah. Good to know. Moved them both.